repo/packages/C/confluent/3/config.json

```diff
@@ -0,0 +1,571 @@
+{
+  "type":"object",
+    "properties":{
+      "service":{
+        "type":"object",
+        "properties":{
+          "name" : {
+            "description":"The name of the Confluent Platform service instance",
+            "type":"string",
+            "default":"confluent"
+          },
+          "placement_strategy":{
+            "description":"Broker placement strategy. See documentation. [ANY, NODE]",
+            "type":"string",
+            "default":"NODE"
+          },
+          "phase_strategy":{
+            "description":"Broker rollout strategy. See documentation. [INSTALL, STAGE]",
+            "type":"string",
+            "default":"INSTALL"
+          }
+        },
+        "required":[
+          "placement_strategy",
+          "phase_strategy"
+        ]
+      },
+      "brokers":{
+        "description":"Kafka broker sizing properties",
+        "type":"object",
+        "properties":{
+          "cpus":{
+            "description":"Broker cpu requirements",
+            "type":"number",
+            "default":1.0
+          },
+          "mem":{
+            "description":"Broker mem requirements",
+            "type":"integer",
+            "default":2048
+          },
+          "disk":{
+            "description":"Broker disk requirements (only respected with persistent volumes)",
+            "type":"integer",
+            "default":5000
+          },

```

Comment:
Could you insert the following block here? The latest framework supports storing data either on a ROOT volume or on a MOUNT volume, which can be configured via this new parameter:

```
          "disk_type": {
            "type": "string",
            "description": "Disk type to be used for storing broker data. See documentation. [ROOT, MOUNT]",
            "default": "ROOT"
          },
```

See also: https://github.com/mesosphere/universe-1.7/blob/version-2.x/repo/packages/K/kafka/15/config.json#L48
Some details on MOUNT vs ROOT volumes here: http://mesos.apache.org/documentation/latest/multiple-disk/